app/components/domain-form.tsx

```diff
@@ -0,0 +1,51 @@
+import { AddIcon, InfoIcon } from '@chakra-ui/icons';
+import { Button, HStack, Input, Select, Textarea, Tooltip, VStack } from '@chakra-ui/react';
+import { Form } from '@remix-run/react';
+
+export default function DomainForm() {
```

Comment:
If I recall from one of the previous discussion we'll have the container centralized, but have its content left aligned?

Reply:
I think the idea was to leave a margin on the left but no margin on mobile. I will try to make it work with your suggestion

Reply:
Hmmm, I could add `Center` but it doesn't affect the page at all. I have added a margin to the left for certain screen sizes. For smaller screens, the margin disappears so I don't think it is worth adding a `Center` here anymore.